pallets/loans/src/functions.rs

```diff
@@ -766,4 +767,35 @@ impl<T: Config> Pallet<T> {
 			},
 		)
 	}
+
+	pub fn get_max_borrow_amount(pool_id: PoolIdOf<T>) -> Result<T::Balance, DispatchError> {
+		let now = Self::now();
+
+		let sum: T::Balance = Zero::zero();
+
+		let active_loans = ActiveLoans::<T>::get(pool_id);
+		for active_loan in active_loans.iter() {
```

Comment:
Using `.map()` didn't work out in a nice way, so I figure a `foreach` iss not the worst solution.

Reply:
`.fold()` is probably the right iterator method here, but I think the for loop is fine